### PR TITLE
Fix the swagger files generated by removing troublesome class

### PR DESCRIPTION
**Pull request by srkukarni**
### Motivation
Function/Source/Sink get returns Function/Source/SinkConfig. Corrected the classname. This dramatically reduces the swagger defn generated for functions/sinks/sources.
Also removed a generated file swagger.json from sourcecode.
### Modifications

*Describe the modifications you've done.*

### Verifying this change

- [ ] Make sure that the change passes the CI checks.

*(Please pick either of the following options)*

This change is a trivial rework / code cleanup without any test coverage.

*(or)*

This change is already covered by existing tests, such as *(please describe tests)*.

*(or)*

This change added tests and can be verified as follows:

*(example:)*
  - *Added integration tests for end-to-end deployment with large payloads (10MB)*
  - *Extended integration test for recovery after broker failure*

### Does this pull request potentially affect one of the following parts:

*If `yes` was chosen, please highlight the changes*

  - Dependencies (does it add or upgrade a dependency): (yes / no)
  - The public API: (yes / no)
  - The schema: (yes / no / don't know)
  - The default values of configurations: (yes / no)
  - The wire protocol: (yes / no)
  - The rest endpoints: (yes / no)
  - The admin cli options: (yes / no)
  - Anything that affects deployment: (yes / no / don't know)

### Documentation

  - Does this pull request introduce a new feature? (yes / no)
  - If yes, how is the feature documented? (not applicable / docs / JavaDocs / not documented)
  - If a feature is not applicable for documentation, explain why?
  - If a feature is not documented yet in this PR, please create a followup issue for adding the documentation

### CODE REVIEW

File: site2/tools/python-doc-gen.sh [hunk 1]
```diff
@@ -26,6 +26,8 @@ ROOT_DIR=$(git rev-parse --show-toplevel)
 # so that Pdoc can import the module
 find $ROOT_DIR -name CMakeCache.txt | xargs rm -f
 find $ROOT_DIR -name CMakeFiles | xargs rm -rf
+find $ROOT_DIR -name PulsarApi.pb.h | xargs rm -rf
```
Comment (sijie) [inline site2/tools/python-doc-gen.sh]: are these 2 lines related to swagger?

Comment (srkukarni) [reply site2/tools/python-doc-gen.sh]: Not directly related to swagger, but these impede the building of the website.

**Comment by srkukarni** [comment]
rerun cpp tests

**Comment by srkukarni** [comment]
rerun java8 tests

**Comment by srkukarni** [comment]
rerun java8 tests
